components/settings/Tabs.tsx

```diff
@@ -2,9 +2,9 @@ import Link from "next/link";
 import { useRouter } from "next/router";
 import { clsx } from "clsx";
 
-const Tabs = ({ active }) => {
+const Tabs = ({ active, tabData }) => {
```

Comment:
```suggestion
const Tabs = ({ active, options }) => {
```
Please change this to `options`